### PR TITLE
Got the plasmatoid lung

### PR DESCRIPTION
**Pull request by Azrun**
[feature][add to wiki]
## About the PR 
Plasmatoid will now infect you and then infect your lungs.  Once the disease has infected your lungs you will find oxygen to be problematic to breath and instead you will need to seek out plasma...

## Why's this needed? 
People asked what plasmatoid is, now there is an answer.

## Changelog

```changelog
(u)Azrun
(+)Plasmatoid results in respiratory "issues".  The infection will spread to the lungs and they will no longer function like they used to.
```

### CODE REVIEW

**Comment by ThatFiveGuys** [comment]
Personally feel like these lungs should offer some sort of buff so they may be something you wish to try and obtain instead of something thats only useful to poison people with.

**Comment by Azrun** [comment]
I like that idea, but that currently seems to conflict with others interpretation of what the the ailment would be.

**Comment by Zonespace27** [comment]
Half-plasmamen, I like it

File: code/modules/medical/diseases/plasmatoid.dm [hunk 1]
```diff
@@ -7,3 +7,54 @@
 	recureprob = 15
 	associated_reagent = "liquid plasma"
 	affected_species = list("Monkey", "Human")
+
+/datum/ailment/disease/plasmatoid/stage_act(mob/living/affected_mob, datum/ailment_data/D)
+	if (..())
+		return
+	switch(D.stage)
+		if(2)
+			if(prob(1))
```
Comment (Zonespace27) [inline code/modules/medical/diseases/plasmatoid.dm:16]: A few prob(1)s doesn't really feel like much in the way of a warning, might be different in practice though

File: code/modules/medical/diseases/plasmatoid.dm [hunk 2]
```diff
@@ -7,3 +7,54 @@
...
+				affected_mob.emote("cough")
+			if(prob(1))
+				boutput(affected_mob, "<span class='alert'>Your muscles ache.</span>")
```
Comment (Zonespace27) [inline code/modules/medical/diseases/plasmatoid.dm:19]: Messages feel fairly unrelated to your lungs being forcefully replaced by a foreign virus

Comment (Azrun) [reply code/modules/medical/diseases/plasmatoid.dm]: Made some updates, felt like it was more of a transformation but at least added some feedback for when it happens.

File: code/modules/medical/diseases/plasmatoid.dm (hunk 1 above)
Comment (Azrun) [reply code/modules/medical/diseases/plasmatoid.dm:19]: Its prob() all the way down....

I restructured so you are more likely to get some symptom but it is possible to make it all the way without a single symptom.

**Comment by flappybatpal** [comment]
Can you breath plasma from a tank to survive with your new plasma lungs?

**Comment by Azrun** [comment]
> Can you breath plasma from a tank to survive with your new plasma lungs?

Yep! That is how I got my spessman to stop being a mess of gasping for air that no longer benefits then.  Sweet sweet mystical plasma.

**Pull request merged**